PokemonAPI/PokemonAPI/NLog.config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      autoReload="true"
+      throwExceptions="false"
+      internalLogLevel="Off" internalLogFile="C:\Users\prome\Desktop\NetCoreWebAPICourse\PokemonAPI\PokemonAPI\Logs\nlog-internal.log">
+
+	<targets>
+		<target xsi:type="File" name="fileTarget" filename="C:\Users\prome\Desktop\NetCoreWebAPICourse\PokemonAPI\PokemonAPI\Logs\log.txt"></target>
+	</targets>
+
+	<rules>
+		<logger name="*" minlevel="Debug" writeTo="fileTarget" />
+
+		<!--Skip non-critical Microsoft logs and so log only own logs-->
+		<logger name="Microsoft.*" maxlevel="Info" final="true" />
+		<!-- BlackHole without writeTo -->
+		<logger name="*" minlevel="Trace" writeTo="ownFile-web" />
```

Comment:
what is ownFile-web

That isn't defined anywhere as **target**

This smells like copy and paste :D